documentation/content/en/books/arch-handbook/boot/_index.adoc

```diff
@@ -901,21 +897,21 @@ The main task for the loader is to boot the kernel. When the kernel is loaded in
 
 [.programlisting]
 ....
-sys/boot/common/boot.c:
+stand/common/boot.c:
     /* Call the exec handler from the loader matching the kernel */
-    module_formats[km->m_loader]->l_exec(km);
+    file_formats[fp->f_loader]->l_exec(fp);
 ....
 
 [[boot-kernel]]
 == Kernel Initialization
 
-Let us take a look at the command that links the kernel. This will help identify the exact location where the loader passes execution to the kernel. This location is the kernel's actual entry point.
+Let us take a look at the command that links the kernel. This will help identify the exact location where the loader passes execution to the kernel. This location is the kernel's actual entry point. This command is now excluded from [.filename]#sys/conf/Makefile.i386#. The content that interests us can be found in [.filename]#/usr/obj/usr/src/i386.i386/sys/GENERIC/#.
```

Comment:
Since you are adding content here, I'd do new sentences per line (unlike above where it's pre-existing structure and you're just changing relatively minor details)

Reply:
I plan on doing a pass of one sentence per line before I push this into the docs tree, after it's been approved.

Reply:
OK. Once you are happy with the formatting, I'm happy (enough) with the content. It updates what's there to be correct, so that's progress.

Reply:
**Thank you very much!** I am very happy!
I am waiting for your final answers and suggestions and I am approaching the final correction.